snowpack/src/types.ts

```diff
@@ -225,17 +225,17 @@ export interface RouteConfigObject {
   _srcRegex: RegExp;
 }
 
-export interface PackageSourceLocal
+export interface PackageOptionsLocal
   extends Omit<
     EsinstallOptions,
     'alias' | 'dest' | 'sourcemap' | 'verbose' | 'logger' | 'cwd' | 'dest' | 'treeshake'
   > {
-  source: 'local';
+  source: 'local' | 'remote-next' | {[key: string]: string};
```

Comment:
WDYT about `npm` or `npm-streaming` instead? I could also see the possibility for a more GitHub-oriented source in the future too.

Reply:
ooooo `npm` is nice. Might be a bit confusing ('isn't local from npm as well') but I do see what you're saying. hmm

Reply:
I'll keep as "remote-next" for now to make the distinction/connection with "remote", but agree that in v4.0 that new name might make more sense once old remote is no longer available.

Reply:
With imports being streamed from npm now, does this mean the requirement of a specific API response from the origin is removed? If so, could source be "stream"? I am imagining where users can define their own endpoint for private packages, not on npm etc.

"private registry support" - Is there a way to specify specific origins for individual deps while streaming?